src/types/navigation.ts

```diff
@@ -17,9 +17,28 @@ export type NavigationItemExternal = NavigationItem & {
   };
 };
 
+export type NavigationItemExtraFilter = {
+  include?: number[];
+  exclude?: number[];
+};
+
+export type NavigationItemAttributeExtraFilter = NavigationItemExtraFilter & {
+  attribute: {
+    id: number;
+    key: string;
+    label: string;
+    type: string;
+    multiSelect: boolean;
+  };
+};
+
 export type NavigationItemCategory = NavigationItem & {
   type: 'category';
-  extraFilters: {[key: string]: {include: number[]}}[];
+  extraFilters: {
+    [key: string]:
+      | NavigationItemExtraFilter
+      | NavigationItemAttributeExtraFilter[];
```

Comment:
I don't think this type is correct if I look at the latest structure:

```
"extraFilters": {
              "merchants": {
                "include": [
                  1
                ]
              },
              "attributes": [
                {
                  "attribute": {
                    "id": 1105,
                    "key": "heelHeightGroup",
                    "label": "Absatzhöhengruppe",
                    "type": "",
                    "multiSelect": false
                  },
                  "include": [
                    1454,
                    1463,
                    1478,
                    1427
                  ]
                }
              ],
              "product_ids": {
                "include": [
                  "41",
                  "38",
                  "12"
                ]
              }
            },
```

Reply:
It looks fine to me 🤔 

I also pinged Chintu if there's a better solution and the answer was basically no